Kentico.Kontent.Delivery.Abstractions/SharedModels/IError.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Kentico.Kontent.Delivery.Abstractions
```

Comment:
generally, try to make the comments a bit more useful (and add the right articles the/a) :)
this has a crucial impact on the developer's experience...when they run into errors they'll appreciate any help narrowing down the root cause.

Reply:
Noted